applications/asset_tracker_v2/src/modules/atv2_led_module.h

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/**@file
+ *
+ * @brief   LED module.
+ *
+ * Module that handles LED behaviour.
+ */
+
+#ifndef LED_H__
+#define LED_H__
+
+#include <zephyr.h>
+#include <caf/led_effect.h>
+
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+enum led_id {
+	LED_1,
+	LED_2,
+	LED_3,
+	LED_4,
+
+	LED_ID_COUNT
+};
+
+
```

Comment:
remove excessive lines
when possible try to keep number of empty lines to minimum; two line separation between different blocks should be enough